custom_typings/yeoman-environment.d.ts

```diff
@@ -4,7 +4,7 @@ declare module 'yeoman-environment' {
     registerStub(generator: any, namespace: string);
     run(target: string, options: any, done: Function);
     getGeneratorsMeta(): YeomanEnvironment.GeneratorMeta[];
-    lookup(callback: () => void);
+    lookup(callback: (error?: Error) => void);
```

Comment:
Be careful about aspirationally strict types in typings for external code. Unless we're confident that this is part of the contract for yeoman-environment, this should probably be typed as `any`